app/views/users/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for @user do |f| %>
+<%= form_for @user, url: yield(:action_path) do |f| %>

```

Comment:
Em, I have a question. What is the purpose of using yield here?

Reply:
edit.html.erb: `<% provide :action_path, user_path %>`
new.html.erb : `<% provide :action_path, signup_path %>`
2 function user same form. i set action_path and use "yield" to reuse form =)))
my english very bad, i hope u understand me :))

Reply:
@iamanhquan @iamanhquan Em, no need to provide the action url dynamically here. you just render the form(from edit and new file) and pass the @user object. No need to worry about the action here, you can inspect the page(developer mode) and see the action in html

Reply:
That's clear, thank you.